packages/remix-forms/src/createForm.tsx

```diff
@@ -262,9 +262,9 @@ function createForm({
 
     const fieldErrors = (key: keyof SchemaType) => {
       const message = (formErrors[key] as unknown as FieldError)?.message
-      return (message && [message]) || (errors && errors[key])
+      return browser() ? (message && [message]) : (errors && errors[key])
```

Comment:
How about the case when the user sets the `errors` prop on the Form? 
I think it will both be in the browser and have this `errors[key]`

Reply:
@felipefreitag, in that case, we would have set the form error in the client [here](https://github.com/SeasonedSoftware/remix-forms/pull/126/files#diff-4931f9de22c9adebebaa2419d630136b4c8bf6f8ebfefb8712adc3697b6e5018R422). We get the error from the prop, action data or fetcher data and set it on the client.

The only reason we still return `(errors && errors[key])` is for when JS is disabled.